### ISSUE TITLE
Cannot predict with TransformerQA on 1.0.0rc5

### ISSUE THREAD

**Issue by schmmd** [allenai/allennlp#4321]
```
$ echo '{"passage": "The Matrix is a 1999 science fiction action film written and directed by The Wachowskis, starring Keanu Reeves, Laurence Fishburne, Carrie-Anne Moss, Hugo Weaving, and Joe Pantoliano.", "question": "Who stars in The Matrix?"}' | allennlp predict https://storage.googleapis.com/allennlp-public-models/transformer-qa-2020-05-26.tar.gz -

2020-06-04 14:15:44,818 - INFO - transformers.file_utils - PyTorch version 1.5.0 available.
2020-06-04 14:15:45,477 - INFO - allennlp.models.archival - loading archive file https://storage.googleapis.com/allennlp-public-models/transformer-qa-2020-05-26.tar.gz from cache at /home/michaels/.allennlp/cache/4c6eacd3c5ba190ae88644f866eb35b9e6ca10b01c15848f166fdb1b020d8a35.6bb2b04ba1dc0eb8d7e4172e5d8c72551fe73b45f947d390ba43ed25d9cce60f
2020-06-04 14:15:45,478 - INFO - allennlp.models.archival - extracting archive file /home/michaels/.allennlp/cache/4c6eacd3c5ba190ae88644f866eb35b9e6ca10b01c15848f166fdb1b020d8a35.6bb2b04ba1dc0eb8d7e4172e5d8c72551fe73b45f947d390ba43ed25d9cce60f to temp dir /tmp/tmpkl0n2lie
2020-06-04 14:15:48,296 - INFO - allennlp.common.params - type = from_instances
2020-06-04 14:15:48,296 - INFO - allennlp.data.vocabulary - Loading token dictionary from /tmp/tmpkl0n2lie/vocabulary.
2020-06-04 14:15:48,296 - INFO - allennlp.common.params - model.type = transformer_qa
2020-06-04 14:15:48,296 - INFO - allennlp.common.params - model.regularizer = None
2020-06-04 14:15:48,296 - INFO - allennlp.common.params - model.transformer_model_name = bert-base-cased
2020-06-04 14:15:48,613 - INFO - transformers.configuration_utils - loading configuration file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-cased-config.json from cache at /home/michaels/.cache/torch/transformers/b945b69218e98b3e2c95acf911789741307dec43c698d35fad11c1ae28bda352.9da767be51e1327499df13488672789394e2ca38b877837e52618a67d7002391
2020-06-04 14:15:48,614 - INFO - transformers.configuration_utils - Model config BertConfig {
  "architectures": [
    "BertForMaskedLM"
  ],
  "attention_probs_dropout_prob": 0.1,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 512,
  "model_type": "bert",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "type_vocab_size": 2,
  "vocab_size": 28996
}

...

2020-06-04 14:15:55,391 - INFO - transformers.tokenization_utils - loading file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-cased-vocab.txt from cache at /home/michaels/.cache/torch/transformers/5e8a2b4893d13790ed4150ca1906be5f7a03d6c4ddf62296c383f6db42814db2.e13dbb970cb325137104fb2e5f36fe865f27746c6b526f6352861b1980eb80b1
Traceback (most recent call last):
  File "/home/michaels/miniconda2/envs/allennlp-rc5/bin/allennlp", line 8, in <module>
    sys.exit(run())
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/__main__.py", line 19, in run
    main(prog="allennlp")
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/commands/__init__.py", line 92, in main
    args.func(args)
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/commands/predict.py", line 212, in _predict
    manager.run()
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/commands/predict.py", line 186, in run
    for model_input_json, result in zip(batch_json, self._predict_json(batch_json)):
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/commands/predict.py", line 132, in _predict_json
    results = [self._predictor.predict_json(batch_data[0])]
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/predictors/predictor.py", line 47, in predict_json
    instance = self._json_to_instance(inputs)
  File "/home/michaels/miniconda2/envs/allennlp-rc5/lib/python3.7/site-packages/allennlp/predictors/predictor.py", line 195, in _json_to_instance
    raise NotImplementedError
NotImplementedError
2020-06-04 14:15:55,419 - INFO - allennlp.models.archival - removing temporary unarchived model dir at /tmp/tmpkl0n2lie
```

**Comment by epwalsh** [comment]
This is due to this: https://github.com/allenai/allennlp-models/blob/master/allennlp_models/rc/predictors/transformer_qa.py#L49

**Comment by schmmd** [comment]
Is it reasonable to consider it a bug that TransformerQA won't work with the CLI @epwalsh ?

**Comment by epwalsh** [comment]
Someone who's more familiar with TransformerQA should probably weigh in. I'm actually not sure why this model creates multiple instances per input.

**Comment by matt-gardner** [comment]
Yes, this should definitely work.  Something needs to be fixed here.

**Comment by dirkgr** [comment]
It's just missing the default predictor. Not sure how that got lost. I added them all when I changed how default predictors are found.

**Comment by schmmd** [comment]
Fixed in 1.0.0rc6.
